src/idom/_console/rewrite_camel_case_props.py

```diff
@@ -0,0 +1,88 @@
+from __future__ import annotations
+
+import ast
+import re
+import sys
+from keyword import kwlist
+from pathlib import Path
+
+import click
+
+from idom._console.ast_utils import (
+    ChangedNode,
+    find_element_constructor_usages,
+    rewrite_changed_nodes,
+)
+
+
+CAMEL_CASE_SUB_PATTERN = re.compile(r"(?<!^)(?=[A-Z])")
+
+
+@click.command()
+@click.argument("paths", nargs=-1, type=click.Path(exists=True))
+def rewrite_camel_case_props(paths: list[str]) -> None:
```

Comment:
This has issues. Currently it does not preserve dashes `-`, so `data-success` gets turned into `data_success`.

Reply:
`data_success` will be handled properly. I think I did that explicitly. The latter case is definitely an issue though.